packages/ui/src/server/configs/common.ts

```diff
@@ -58,6 +58,7 @@ const config: Partial<AppConfig> = {
         reHashFromNodeVersion: '[^~]+~(?<hash>[^+]+)',
         directDownload: true,
         docsBaseUrl: process.env.YT_DOCS_BASE_URL || 'https://ytsaurus.tech/docs/en',
+        reUseEffectiveAclForPath: '//sys/access_control_object_namespaces[^/+]{0,}',
```

Comment:
I don't think we need default value for the option here (or in local.ts). But it is a good idea to have corresponding e2e to be sure the functionality works propertly.